### ISSUE TITLE
Text display lagging in the Grade Assignment by Rubric when typing comments for each criterion [Check again after Vue]

### ISSUE THREAD

**Issue by chcholman** [UM-USElab/gradecraft-development#4273]
Feedback from Rebecca:
Often when she was typing text in the Grade Assignment by Rubric section where you describe why you gave them the rubric level you did, it would be laggy. And sometimes SUPER laggy. Text wouldn’t show up as you typed it, but much after.

┆Attachments: <a href="https://trello-attachments.s3.amazonaws.com/5a6f2020d4ad818ef1fffe12/5ce8004ad418857defef0466/a5f6ec0f31485c1a4a6625856895aff9/Grade_Assignment_by_Rubric_text_lagging.png">Grade Assignment by Rubric text lagging.png</a>

**Comment by adishy** [comment]
Couldn't really reproduce this issue in the local environment. Could be a performance bug related to some sort of memory leak on the client side, could also have been caused due to network latency or some other similar issue, but the text box behaved as expected in the local environment and I did not notice any performance drops. I did however notice a suspicious spike in CPU load of about 20-30% as I was typing, don't know for sure whether this is relevant. Any more information about this would be welcome!
